demoqa/src/main/java/framework/SeleniumTestBase.java

```diff
@@ -0,0 +1,39 @@
+package framework;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+public abstract class SeleniumTestBase {
+	private WebDriver driver;
+	
+	@BeforeTest
+	public void setup() {
+	System.setProperty("webdriver.chrome.driver", "C:\\ChromeDriver\\chromedriver.exe");
+	
+	this.driver =  new ChromeDriver();
+	
+	setOptions();
```

Comment:
Remove dead code (this method does nothing)

Reply:
Resolved